README.rst

```diff
@@ -240,3 +246,88 @@ You can find the id and secret of your app at https://developers.facebook.com/ap
 For testing purposes you can use the access token `<user_access_token>` from https://developers.facebook.com/tools/accesstoken/.
 
 For more information on how to configure python-social-auth with Facebook visit http://python-social-auth.readthedocs.io/en/latest/backends/facebook.html.
+
+
+Google Example
+--------------------
+
+To use Google OAuth2 as the authorization backend of your django-rest-framework api, your settings.py file should look like this:
+
+.. code-block:: python
+
+    INSTALLED_APPS = (
+        ...
+        # OAuth
+        'oauth2_provider',
+        'social_django',
+        'rest_framework_social_oauth2',
+    )
+
+    TEMPLATES = [
+        {
+            ...
+            'OPTIONS': {
+                'context_processors': [
+                    ...
+                    # OAuth
+                    'social_django.context_processors.backends',
+                    'social_django.context_processors.login_redirect',
+                ],
+            },
+        }
+    ]
+
+    REST_FRAMEWORK = {
+        ...
+        'DEFAULT_AUTHENTICATION_CLASSES': (
+            ...
+            # OAuth
+            # 'oauth2_provider.ext.rest_framework.OAuth2Authentication',  # django-oauth-toolkit < 1.0.0
+            'oauth2_provider.contrib.rest_framework.OAuth2Authentication',  # django-oauth-toolkit >= 1.0.0
+            'rest_framework_social_oauth2.authentication.SocialAuthentication',
+        )
+    }
+
+    AUTHENTICATION_BACKENDS = (
+
+        # Others auth providers (e.g. Facebook, OpenId, etc)
+        ...
+
+		# Google OAuth2
+		'social_core.backends.google.GoogleOAuth2',
+
+        # django-rest-framework-social-oauth2
+        'rest_framework_social_oauth2.backends.DjangoOAuth2',
+
+        # Django
+        'django.contrib.auth.backends.ModelBackend',
+
+    )
+
+    # Google configuration
+    SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = <your app key goes here>
+	SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = <your app secret goes here>
+
+    # Define SOCIAL_AUTH_GOOGLE_OAUTH2_SCOPE to get extra permissions from Google.
+    SOCIAL_AUTH_GOOGLE_OAUTH2_SCOPE = [
+    'https://www.googleapis.com/auth/userinfo.email',
+    'https://www.googleapis.com/auth/userinfo.profile',
+	]
+
+Remember to add new Application in your Django admin (see section "Setting up Application").
+
+Testing
+*******
+
+You can test these settings by running the following command :
+
+    curl -X POST -d "grant_type=convert_token&client_id=<django-oauth-generated-client_id>&client_secret=<django-oauth-generated-client_secret>&backend=google-oauth2&token=<google_token>" http://localhost:8000/auth/convert-token
+
```

Comment:
Where is <google_token> defined?
Where do I get that value?

Reply:
Good question :) `<google_token>` is returned by Google OAuth service, just like in case of 

> curl -X POST -d "grant_type=convert_token&client_id=<client_id>&client_secret=<client_secret>&backend=facebook&token=<facebook_token>" http://localhost:8000/auth/convert-token

I followed the same pattern as for Facebook example. But I see now naming might be confusing and I'm open to suggestions.

Reply:
I was just playing with it here the other day. I am new to django and drf. 
I think i need to find a tutorial somewhere, i could not get it to work.
Do you have some suggestions? 
btw good job :)

Reply:
@tir-bogo, you mean how to setup DRF and Django to allow Google social auth login? Or just how to setup Django rest framework social oauth2?